showcase/app/routes/page-components/tag.ts

```diff
@@ -14,7 +14,11 @@ import type { ModelFrom } from 'showcase/utils/model-from-route';
 export type PageComponentsTagModel = ModelFrom<PageComponentsTagRoute>;
 
 export default class PageComponentsTagRoute extends Route {
-  model() {
+  model(): {
+    COLORS: typeof COLORS;
+    TOOLTIP_PLACEMENTS: typeof TOOLTIP_PLACEMENTS;
+    STATES: string[];
+  } {
     // these are used only for presentation purpose in the showcase
     const STATES = ['default', 'hover', 'active', 'focus'];
     return { COLORS, TOOLTIP_PLACEMENTS, STATES };
```

Comment:
@shleewhite I thought it was still needed because it contained these states, etc that it returned. Is that not the case?